core/new-gui/src/app/dashboard/component/top-bar/user-icon/user-login/ngbdmodal-user-login.component.ts

```diff
@@ -30,6 +33,24 @@ export class NgbdModalUserLoginComponent implements OnInit {
     this.detectUserChange();
   }
 
+  signupForms = new FormGroup(
+    {
+      password: new FormControl('', [Validators.required]),
+      verifyPassword: new FormControl('', [Validators.required]),
+    },
+  )
+
+  public errorMessageLoginPasswordNull(): string{
+    return this.loginPassword.hasError('required') ? "Password required" : "";
+  }
+
+  public errorMessageRegisterPasswordNull(): string{
+    return this.registerPassword.hasError('required') ? "Password required" : "";
+  }
+
+  public errorMessageRegisterConfirmationPasswordNull(): string{
+    return this.registerConfirmationPassword.hasError('required') ? "Confirmation required" : "";
+  }
```

Comment:
define a single error handler for all different checks and messages

Reply:
Done. Also added an error message when the username field is empty.